src/GraphQL.Analyzers.CodeFixes/FieldBuilderCodeFixProvider.cs

```diff
@@ -0,0 +1,252 @@
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Formatting;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace GraphQL.Analyzers;
+
+[Shared]
+[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(FieldBuilderCodeFixProvider))]
+public class FieldBuilderCodeFixProvider : CodeFixProvider
+{
+    public static string ReformatOption { get; } = $"dotnet_diagnostic.{FieldBuilderAnalyzer.DoNotUseObsoleteFieldMethods.Id}.reformat";
+
+    public static string SkipNullsOption { get; } = $"dotnet_diagnostic.{FieldBuilderAnalyzer.DoNotUseObsoleteFieldMethods.Id}.skip_nulls";
+
+    public override ImmutableArray<string> FixableDiagnosticIds { get; } =
+        ImmutableArray.Create(FieldBuilderAnalyzer.DoNotUseObsoleteFieldMethods.Id);
+
+    public sealed override FixAllProvider GetFixAllProvider() =>
+        WellKnownFixAllProviders.BatchFixer;
+
+    public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+    {
+        var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+
+        foreach (var diagnostic in context.Diagnostics)
+        {
+            var diagnosticSpan = diagnostic.Location.SourceSpan;
+            bool isAsyncField = diagnostic.Properties.ContainsKey(Constants.Properties.IsAsync);
+            bool isDelegate = diagnostic.Properties.ContainsKey(Constants.Properties.IsDelegate);
+
+            var fieldInvocationExpression = (InvocationExpressionSyntax)root!.FindNode(diagnosticSpan);
+
+            const string codeFixTitle = "Rewrite obsolete 'Field' method";
```

Comment:
This is the title that will be shown in the yellow lightbulb as the fix suggestion. I need help with choosing good words, both, for detected diagnostics and fix suggestions.